packages/ci/src/lib/monorepo/list-projects.ts

```diff
@@ -95,16 +96,16 @@ function createMonorepoHandlerOptions(
     cwd: settings.directory,
     parallel: settings.parallel,
     nxProjectsFilter: settings.nxProjectsFilter,
-    ...(!settings.silent && {
-      observer: {
+    observer: {
+      onStderr: stderr => {
+        console.warn(stderr);
+      },
+      ...((!settings.silent || isVerbose()) && {
         onStdout: stdout => {
           console.info(stdout);
         },
-        onStderr: stderr => {
-          console.warn(stderr);
-        },
-      },
-    }),
+      }),
+    },
```

Comment:
This logic is duplicated in `createCommandContext`. How about extracting the observer creation?